TypeCobol.LanguageServer/Commands/AdjustFillers.cs

```diff
@@ -105,6 +109,17 @@ void AddOrAdjust()
                     {
                         textEdits.Add(textEdit);
                     }
+
+                    DataDefinition GetLastDescendant(DataDefinition dataDefinition)
+                    {
+                        if (HasNoChildrenExcludingIndex(dataDefinition))
+                        {
+                            return dataDefinition;
+                        }
+
+                        Debug.Assert(dataDefinition.Children[^1] is DataDefinition);
+                        return GetLastDescendant((DataDefinition)dataDefinition.Children[^1]);
+                    }
```

Comment:
No need to bother with the node's type here. I agree that it should be data definitions all the way but I'm worried with EXEC nodes and some ill-formed AST.

```csharp
Node GetLastNode(Node node)
{
    if (HasNoChildrenExcludingIndex(node))
    {
        return node;
    }

    return GetLastNode(node.Children[^1]);
}
```

should work as well

Reply:
Also there is a `GetNextNode` method on `Node` object, so this new helper could be moved there for future uses.

Reply:
Done